open3d_conversions/include/open3d_conversions/open3d_conversions.hpp

```diff
@@ -43,6 +45,38 @@ void open3dToRos(
   const open3d::geometry::PointCloud & pointcloud,
   sensor_msgs::msg::PointCloud2 & ros_pc2,
   std::string frame_id = "open3d_pointcloud");
+
+/**
+ * @brief Copy data from a open3d::geometry::Image to a 
+ * sensor_msgs::msg::Image
+ * 
+ * @param image Reference to the open3d geometry Image
+ * @param ros_img Reference to the sensor_msgs Image
+ * @param encoding The encoding of the image to use in ROS format
+ * @param frame_id The string to be placed in the frame_id of the Image
+ */
+void open3dToRos(
+  const open3d::geometry::Image & o3d_img,
+  sensor_msgs::msg::Image & ros_img,
+  std::string encoding,
+  std::string frame_id = "open3d_image");
```

Comment:
I'm not sure there's any rational default for this, I think it should be a required field (for all)

Reply:
I agree. I had mainly been trying to match the functions that were already existed. My thoughts are to remove that parameter entirely since a user would still have to assign the header timestamp field manually, so they might as well assign the frame_id manually as well.